### ISSUE TITLE
Multi-page TIFF file support?

### ISSUE THREAD

**Issue by vatterspun** [kanryu/quickviewer#39]
My 7zip bug seems on the outs so I tried to do something else useful...

Setup: Win7x32

Steps:

1. Create a mutli-page TIFF file using PDFXChange Editor (just convert something over from PDF)
2. Convert to TIFF format

![image](https://user-images.githubusercontent.com/1289134/30625233-a4563e54-9d87-11e7-8d0f-6418236b4561.png)

3. Open in Quickviewer (even with 2-page spread enabled)

![image](https://user-images.githubusercontent.com/1289134/30625279-e90365e0-9d87-11e7-8e0c-ec4c347bdf0a.png)

Defect: only the first page is visible.

**Comment by kanryu** [comment]
First of all, since Multipage tiff and Windows Ico are not archives, we do not plan to support image switching.

I tried to open Multipage Tiff on this page, but imageformats / qtiff.dll failed to open. 
http://www.nightprogrammer.org/development/multipage-tiff-example-download-test-image-file/

And if I try to open it again after failing it will end incorrectly.
It is awkward because I did not expect such behavior.

In order to solve this problem you will have to make yourself an imageformat plugin that can accurately read complex Tiff file includes multiple JPEGs.

Currently it is not considered to positively use Tiff format images, so I think that it is better to consider converting to other image formats.
